src/applications/mhv-secure-messaging/components/ComposeForm/RecipientsSelect.jsx

```diff
@@ -78,13 +78,21 @@ const RecipientsSelect = ({
 
   const handleRecipientSelect = useCallback(
     e => {
-      const recipient = recipientsList.find(r => +r.id === +e.detail.value);
-      setSelectedRecipient(recipient);
-      if (recipient.signatureRequired || isSignatureRequired) {
+      const recipient = recipientsList?.find(r => +r.id === +e.detail.value);
+
+      if (e.detail.value === '') {
+        setSelectedRecipient({});
+      }
+
+      if (e.detail.value !== '') {
+        setSelectedRecipient(recipient);
+      }
+
+      if (recipient?.signatureRequired || isSignatureRequired) {
         setAlertDisplayed(true);
       }
```

Comment:
a tiny suggestion to potentially make the selectedRecipient easier to follow:

```suggestion
      if (!+e.detail.value) {
      setSelectedRecipient({});
      return;
    }

    const recipient = recipientsList?.find(r => +r.id === +e.detail.value) || {};
    setSelectedRecipient(selected);

    if (recipient.signatureRequired || isSignatureRequired) {
      setAlertDisplayed(true);
    }
```

Reply:
Thanks! Nice suggestion